neptune/new/__init__.py

```diff
@@ -32,7 +32,7 @@
     NeptuneUninitializedException,
     NeptunePossibleLegacyUsageException,
 )
-from neptune.new.run import Run
+from neptune.new.attributes_containers import Run, Project, Model, ModelVersion
```

Comment:
Should we make `*Table` and `*TableEntry` objects importable for our users as well?